packages/ckeditor5-dev-utils/lib/builds/getpluginwebpackconfig.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const path = require( 'path' );
+const webpack = require( 'webpack' );
+const bundler = require( '../bundler' );
+const styles = require( '../styles' );
+const tools = require( '../tools' );
+
+/**
+ * Returns a webpack configuration that creates a bundle file for the specified package. Thanks to that, plugins exported
+ * by the package can be added to ready-to-use builds.
+ *
+ * @param {Object} options
+ * @param {String} options.themePath An absolute path to the theme package.
+ * @param {String} options.packagePath An absolute path to the root directory of the package.
+ * @returns {Object}
+ */
+module.exports = function getPluginWebpackConfig( options ) {
```

Comment:
I'd go with `getDLLPluginWebpackConfig()` just to be clear about that.

Reply:
Fixed in 1eac3f7.